### PR TITLE
add stochss-launcher repo

### PR DESCRIPTION
**Pull request by aviral26**

### CODE REVIEW

**Comment by ahellander** [comment]
Two comments: 
   1. Is it obvious that this should be part of the main app repository? One alternative would be to have it as a separate repo, since it is not really using any of the code in stochss (pulls a pre-made Docker image from Docker hub?) 
 2. I think the .zip archives should not be included with the source, they should be released elsewhere, like on the webpage.

**Comment by briandrawert** [comment]
Addressing Andreas' comments above:
1. This was in a different repo, and I would still be sorta ok with it being separate if we can move the issue list for that repo from Aviral's account to the StochSS account.  I do not believe that is possible.
2. The Zip archives contain the ".app" package.  We definitaly need those ".app"s under version control.  Should they be unzipped and added?  Is there some reason they have to be included as zip files?

**Comment by briandrawert** [comment]
Can you add a "windows" folder, and put a short README file in there on exactly what steps to take?

**Comment by aviral26** [comment]
Brian,

No, there isn’t any particular reason I have them as zipped files in the repo. We could have them as .app files or even a single zip file containing both the .app files.

**Comment by aviral26** [comment]
Yes, I’ll have that done by tomorrow.

**Comment by briandrawert** [comment]
Ok thanks! Since zip files do not give fine grained version control, let's add the .app folders to the repo.

**Comment by aviral26** [comment]
Okay, I’ll update the repo.

**Comment by aviral26** [comment]
Should I add READMEs for the Linux and OSX folders too?

**Comment by briandrawert** [comment]
Yes, READMEs in those folders would be very helpful.

File: stochss-launcher/Linux/launchContainerLinux1.7.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+function clean_up(){
+	echo
+	echo "Stopping StochSS...this may take a while"
+	if [[ $(uname -s) == 'Linux' ]]
+	then
+		(docker stop stochsscontainer1_7 || echo "Could not stop container")
+	elif [[ $(uname -s) == 'Darwin' ]]
+	then
+		echo "Not stopping VM while debugging"
```
Comment (briandrawert) [inline stochss-launcher/Linux/launchContainerLinux1.7.sh]: Make sure to remove all debugging lines before we accept the pull.

File: stochss-launcher/OSX/pullImage1.7.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#echo "exporting path $1"
```
Comment (briandrawert) [inline stochss-launcher/OSX/pullImage1.7.sh]: remove

File: run.mac.sh [hunk 3]
```diff
@@ -6,6 +6,34 @@ if [ "$osname" != 'Darwin' ]; then
     exit
 fi
 
+mode=""
```
Comment (aviral26) [inline run.mac.sh]: I have replaced all occurrences of "$1" with "$mode", assuming that "$1" represented the "--debug" argument when passed. I can change this if required.

**Pull request merged**